controllers/progressivesync_controller_test.go

```diff
@@ -32,7 +35,8 @@ const (
 
 var (
 	appSetAPIRef = utils.AppSetAPIGroup
-	ctx          = context.Background()
+	ctx          context.Context
```

Comment:
I am not the greatest fan of globals, especially if is hard to tell that they are even globals because they have the same naming convention as local variables. I cannot find any guidance for Go about naming the global variables, but I'd personally be happier if we could uppercase them or prefix them with `g` like in C or something.